doc/intl.md

```diff
@@ -0,0 +1,196 @@
+
+- `Intl.DateTimeFormat`
+  - `Intl.DateTimeFormat.supportedLocalesOf`
+  - `Intl.DateTimeFormat.prototype.format`
+  - `Intl.DateTimeFormat.prototype.formatToParts`
+  - `Intl.DateTimeFormat.prototype.resolvedOptions`
+
+- `Intl.getCanonicalLocales`
+
+- `String.prototype`
+  - `localeCompare`
+  - `toLocaleLowerCase`
+  - `toLocaleUpperCase`
+
+- `Array.prototype`
+  - `toLocaleString`
+
+- `Number.prototype`
+  - `toLocaleString`
+
+- `Date.prototype`
+  - `toLocaleString`
+  - `toLocaleDateString`
+  - `toLocaleTimeString`
+
+## Not yet supported
+
+- [`Intl.PluralRules`](https://tc39.es/ecma402/#pluralrules-objects)
+
+- [`Intl.RelativeTimeFormat`](https://tc39.es/ecma402/#relativetimeformat-objects)
+
+- [`Intl.DisplayNames`](https://tc39.es/proposal-intl-displaynames/#sec-intl-displaynames-constructor)
+
+- [`Intl.ListFormat`](https://tc39.es/proposal-intl-list-format/#sec-intl-listformat-constructor)
+
+- [`Intl.Locale`](https://tc39.es/ecma402/#sec-intl-locale-constructor)
+
+- `Intl.DateTimeFormat` properties
+   - [`dateStyle/timeStyle`](https://tc39.es/proposal-intl-datetime-style/)
+   - [`dayPeriod`](https://github.com/tc39/ecma402/issues/29)
+   - [`fractionalSecondDigits`](https://github.com/tc39/ecma402/pull/347)
+- [`BigInt.prototype.toLocaleString`](https://tc39.es/ecma402/#sup-bigint.prototype.tolocalestring)
+
+## Excluded
+
+- `Intl.DateTimeFormat`: [`formatMatcher`](https://tc39.es/ecma402/#sec-basicformatmatcher) parameter is not respected. The parameter enables the implementation to pick the best display format when it supports only a subset of all possible formats. ICU library in Android platform and hence our implementation allows all subsets and formats which makes this `formatMatcher` property unnecessary.
+
+## Known Issues
+
+### Android 11
+
+- The keys of the object returned by `resolvedOptions` function in all `Intl` services are not deterministically ordered as prescribed by spec.
+- DateFormat: ECMAScript [beginning of time](https://www.ecma-international.org/ecma-262/11.0/index.html#sec-time-values-and-time-range) (-8,640,000,000,000,000), is formatted as `November 271817`, instead of expected `April 271822`.
+- `Intl.NumberFormat` implementation has some rough edges in supporting the following properties,
+  - `style`: 'unit'
+  - `notation`: 'compact'
+  - `signDisplay`
+  - `currencyFormat`
+
+### Android 10 and older (SDK < 30)
+
+- `Intl.NumberFormat`: Scientific notation formatting has issues on some cases such as Infinity (e.g. Expected SameValue(«-∞E0», «-∞») to be true).
+- `Intl.NumberFormat`: Compact notation `formatToParts` doesn't identify unit, hence we report unit as 'literal' (Compact short: 987654321: parts[1].type Expected SameValue(«literal», «compact») to be true)
+- `Intl.NumberFormat`: `formatToParts` function doesn't produce expected result with Scientific/Engineering notation and input Infinity (`-Infinity` - engineering: length Expected SameValue(«4», «2») to be true).

```

Comment:
Same here, what is "`-Infinity` - engineering:"?

Reply:
This one is strange .. When i was testing a few months back, `-Inifinity` was foamtted as something like `∞E0` on older than platform 30 .. but from a quick test, it doesn't seem to be the case anymore .. May be the icu libraries are getting updated over the air .. I'll keep the entry anyways but make the text clearer.